ci/taos/checker-pr-audit-async.sh

```diff
@@ -230,19 +230,30 @@ popd
 # Job queue: Fairness or FCFS is not guaranteed.
 # $RANDOM is an internal bash function (not a constant) - http://tldp.org/LDP/abs/html/randomvar.html
 # To enhance a job queue, refer to http://hackthology.com/a-job-queue-in-bash.html
-
-# The Pull-Request scheduler to manage the run queues that start gbs build (Tizen)
 # The default RUN Queue is declared in the configuration file
+
```

Comment:
If you intend to check
```
wait util (if # gbs < X && # pdebuild < Y)
```

Why don't you check in a single loop? This serial-loops are going to misbehave soon.
E.g.,
```
current_jobs=$(( $(eval "...") + $( $(eval "...") ))
```

Reply:
The reason and goal is to avoid an unexpected PR hang situation while running the build procedures in AWS instance (KVM-based instance according Amazon engineer). 

>  `# a. This statement has to be put in front of Ubuntu build more than this location`

Please, see Todo  a. to do this wait policy 
First, in front of  gbs build (Tizen)
Second, in front of pdebuild (Ubuntu) <--- Soon.!!!!